### PR TITLE
Feat: Create open channel moderation of operator

### PR DESCRIPTION
**Pull request by HoonBaek**
## For Internal Contributors

[UIKIT-1849](https://sendbird.atlassian.net/browse/UIKIT-1849)

## Description Of Changes

* Refactor component (language translation)
  * Checkbox
  * UserListItem
* Rename (internal usage)
  * ParticipantsList -> ParticipantList
  * BannedMemberList -> BannedUserList
* Create components under the OpenChannelSettings
  * Modal
    * AddOperatorsModal
    * BannedUsersModal
    *  MutedParticipantsModal
    * OperatorsModal
  * User List
    * BannedUserList
    * MutedParticipantList
    * OperatorList
* Add string set
  * OpenChannelModeration
      OPEN_CHANNEL_SETTINGS__OPERATORS_TITLE: 'Operators',
      OPEN_CHANNEL_SETTINGS__OPERATORS__TITLE_ADD: 'Add operator',
      OPEN_CHANNEL_SETTINGS__OPERATORS__TITLE_ALL: 'All operators',
      OPEN_CHANNEL_SETTINGS__MUTED_MEMBERS__TITLE: 'Muted users',
      OPEN_CHANNEL_SETTINGS__MUTED_MEMBERS__TITLE_ALL: 'All muted users',
      OPEN_CHANNEL_SETTINGS__MUTED_MEMBERS__NO_ONE: 'No muted users yet',
      OPEN_CHANNEL_SETTINGS__BANNED_MEMBERS__TITLE: 'Banned users',
      OPEN_CHANNEL_SETTINGS__BANNED_MEMBERS__TITLE_ALL: 'All banned users',
      OPEN_CHANNEL_SETTINGS__BANNED_MEMBERS__NO_ONE: 'No banned users yet',
      OPEN_CHANNEL_SETTINGS__MEMBERS__YOU: ' (You)',
      OPEN_CHANNEL_SETTINGS__MEMBERS__OPERATOR: 'Operator',
      OPEN_CHANNEL_SETTING__MODERATION__REGISTER_AS_OPERATOR: 'Register as operator',
      OPEN_CHANNEL_SETTING__MODERATION__UNREGISTER_OPERATOR: 'Unregister operator',
      OPEN_CHANNEL_SETTING__MODERATION__MUTE: 'Mute',
      OPEN_CHANNEL_SETTING__MODERATION__UNMUTE: 'Unmute',
      OPEN_CHANNEL_SETTING__MODERATION__BAN: 'Ban',
      OPEN_CHANNEL_SETTING__MODERATION__UNBAN: 'Unban',
  * GroupChannelModeration
      CHANNEL_SETTING__MEMBERS__OPERATOR: 'Operator', 
      CHANNEL_SETTING__MODERATION__REGISTER_AS_OPERATOR: 'Register as operator',
      CHANNEL_SETTING__MODERATION__UNREGISTER_OPERATOR: 'Unregister operator',
      CHANNEL_SETTING__MODERATION__MUTE: 'Mute',
      CHANNEL_SETTING__MODERATION__UNMUTE: 'Unmute',
      CHANNEL_SETTING__MODERATION__BAN: 'Ban',
      CHANNEL_SETTING__MODERATION__UNBAN: 'Unban',
    * change
      CHANNEL_SETTING__BANNED_MEMBERS__TITLE: 'Banned users', `(members -> users)`

## Types Of Changes

What types of changes does your code introduce to this project?
Put an `x` in the boxes that apply_

- [ ] Bugfix
- [x] New feature
- [ ] Documentation (correction or otherwise)
- [ ] Cosmetics (whitespace, appearance (ex) Prettier)
- [ ] Build configuration
- [x] Improvement (refactor code)

### CODE REVIEW

File: src/smart-components/ChannelSettings/components/ModerationPanel/MembersModal.tsx [hunk 1]
```diff
@@ -173,7 +184,7 @@ export default function MembersModal({ onCancel }: Props): ReactElement {
                               });
                             }}
                           >
-                            Ban
+                            {stringSet.CHANNEL_SETTING__MODERATION__BAN}
```
Comment (sravan-s) [inline src/smart-components/ChannelSettings/components/ModerationPanel/MembersModal.tsx:187]: 👍

File: src/smart-components/ChannelSettings/components/ModerationPanel/MemberList.tsx [hunk 2]
```diff
@@ -103,18 +99,22 @@ export const MemberList = (): ReactElement => {
                           onClick={() => {
                             if ((member.role !== 'operator')) {
                               channel?.addOperators([member.userId]).then(() => {
-                                refershList();
+                                refreshList();
```
Comment (sravan-s) [inline src/smart-components/ChannelSettings/components/ModerationPanel/MemberList.tsx:102]: 👍

File: src/smart-components/OpenChannelSettings/components/OperatorUI/AddOperatorsModal.tsx [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    });
+    participantListQuery.next().then((users) => {
+      setParticipants(users);
+    });
+    setParticipantQuery(participantListQuery);
+  }, [])
+
+  const selectedCount = Object.keys(selectedUsers).filter((m) => selectedUsers[m]).length;
+  return (
+    <div>
```
Comment (sravan-s) [inline src/smart-components/OpenChannelSettings/components/OperatorUI/AddOperatorsModal.tsx:46]: Probably dont need this div here

File: src/smart-components/OpenChannelSettings/components/OperatorUI/OperatorsModal.tsx [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+    participantListQuery.next().then((participants) => {
+      setUsers(participants);
+    });
+    setOperatorQuery(participantListQuery);
+  }, [])
+  return (
+    <div>
+      <Modal
+        hideFooter
+        titleText="All users"
```
Comment (sravan-s) [inline src/smart-components/OpenChannelSettings/components/OperatorUI/OperatorsModal.tsx:42]: Will be good to localize `All users` (in other files also)

**Review by sravan-s — APPROVED** [review]
LGTM
Please merge next week ~

File: src/smart-components/OpenChannelSettings/components/OperatorUI/OperatorsModal.tsx (hunk 4 above)
Comment (HoonBaek) [reply src/smart-components/OpenChannelSettings/components/OperatorUI/OperatorsModal.tsx]: Let's do that with string set task

**Pull request merged**